custom/conf/app.example.ini

```diff
@@ -938,6 +938,14 @@ LEVEL = Info
 
 ;; Allow fork repositories without maximum number limit
 ;ALLOW_FORK_WITHOUT_MAXIMUM_LIMIT = true
+;
+;; Enable applying a global size limit defined by REPO_SIZE_LIMIT. Each repository can have a value that overrides the global limit
+;; "false" means no limit will be enforced, even if specified on a repository
+;ENABLE_SIZE_LIMIT = false
+;
+;; Specify a global repository size limit to apply for each repository. 0 - No limit
+;; If repository has it's own limit set in UI it will override the global setting
+;REPO_SIZE_LIMIT = 0
```

Comment:
What's the number's meaning? byte ?

Reply:
Hi, yes, byte. Added it here. Also added the units of measurement samples that can be used with this value. 
Please if ok, resolve (or is it only me who can do it?)

Reply:
Could you add the size unit description in the comment?